dataactbroker/handlers/submission_handler.py

```diff
@@ -85,6 +87,65 @@ def get_submission_stats(submission_id):
     }
 
 
+def get_submission_metadata(submission):
+    """ Get metadata for the submission specified
+
+        Args:
+            submission: submission to retrieve metadata for
+
+        Returns:
+            object containing metadata for the submission
+    """
+    sess = GlobalDB.db().session
+
+    # Determine the agency name
+    cgac = sess.query(CGAC).filter_by(cgac_code=submission.cgac_code).one_or_none()
+    frec = sess.query(FREC).filter_by(frec_code=submission.frec_code).one_or_none()
```

Comment:
Do we want to call frec when we don't have to? Instead of assigning it to a variable we could call it within the conditional.